lib/ansible_tower_client/job_template.rb

```diff
@@ -27,5 +34,28 @@ def survey_spec
     def self.endpoint
       "job_templates".freeze
     end
+
+    private
+
+    def with_temporary_changes(limit)
+      old_limit = @raw_body['limit']
+      new_limit = limit
+      patch("{ \"limit\": \"#{new_limit}\" }")
+      begin
+        yield
+      rescue => e
```

Comment:
I was going to ask about that - I didn't feel great about leaving `rescue` open ended either but wasn't sure on the standard operating procedure.